manifests/server/passenger.pp

```diff
@@ -42,11 +44,21 @@
     'path'              => "${app_root}/public/",
     'passenger_enabled' => 'On',
   }
-  
+
   $directories = [
     $directory,
   ]
 
+  $http_pre_start = $passenger_pre_start ? {
+    true  => "http://${::fqdn}:${http_port}",
```

Comment:
Without the / should be finde, see https://github.com/theforeman/puppet-foreman/blob/76c7f4bb52383458ba6abc49a02c049e067986f3/manifests/config/passenger.pp#L160